functionality/sub.py

```diff
@@ -0,0 +1,40 @@
+import paho.mqtt.client as mqtt
+import time as time
+import json
+
+user = "user"
+password = "password"
+topic_1 = "awsiot_to_localgateway"
+topic_2 = "both_directions"
+broker_address="110.174.81.168"
```

Comment:
store IP & port in configuration/ constant file?

Reply:
I will create a json file with all topics